README.md

```diff
@@ -76,6 +70,14 @@ Note that this expects a standard setup with:
 
 When working with `docker compose`, it's generally recommended to avoid committing changes to the `docker-compose.dev.yml` file, especially if you're running the `./load_mongodump <dump tar>` script. This file is typically configured to work in a specific way for your development environment, and changes might not be applicable or useful for others working on the same project.
 
+If you feel the need to adjust the Dockerfiles or any scripts pertaining to the containers, you must rebuild the images with:
+
+```bash
+docker-compose -f docker-compose.dev.yml build
+```
```

Comment:
This is not true. It is true for the production version which copies the files into the container, but not for the dev version, which just maps `viz_scripts` as a volume mount.

Reply:
ok! I have added this clarification.